esbuild.mjs

```diff
@@ -0,0 +1,80 @@
+import glob from 'glob';
+import { build } from 'esbuild';
+import postCssPlugin from "./esbuild.postcss.js";
+import svgr from 'esbuild-plugin-svgr';
+import syntax from 'postcss-scss';
+import cssImport from 'postcss-import';
+import cssStripComments from 'postcss-strip-inline-comments';
+import precss from 'precss';
+import cssFunctions from 'postcss-functions';
+import cssUtil from './util/index.js';
+import cssUrl from 'postcss-url';
+import cssFocusVisible from 'postcss-focus-visible';
+import tailwindcss from 'tailwindcss';
+import autoprefixer from 'autoprefixer';
+import cssnano from 'cssnano';
+import purgecss from '@fullhuman/postcss-purgecss';
+
+
+const entryPoints = glob.sync('./src/**/!(*documentation)/**/*[!*.d].{ts,tsx,css}');
+
+const options = {
+  entryPoints,
+  outdir: './dist',
+  outbase: './src',
+  format: 'esm',
+  tsconfig: 'tsconfig.json',
+  logLevel: 'info',
+  minify: true,
+  sourcemap: true,
+  // bundle: true,
+  plugins: [
+    postCssPlugin({
+      inject: false,
+      syntax,
+      plugins: [cssImport, cssStripComments, precss, cssFunctions({ functions: { hexToRgb: cssUtil.hexToRGB } }),
+        cssUrl({
+          url: 'copy',
+          maxSize: 10 * 1024, // inline files < 10k, copy files > 10k
+          fallback: 'copy',
+          optimizeSvgEncode: true,
+          assetsPath: 'dist/assets',
+        }),
+        cssFocusVisible,
+        tailwindcss,
+        autoprefixer,
+        cssnano({
+          preset: ['default', {
+            discardComments: {
+              removeAll: true,
+            },
+          }]
+        }),
+        purgecss({
+          content: ['./src/**/!(*documentation)/**/*[!*.d].{ts,tsx}']
+        })
+      ]
+    }),
```

Comment:
Why do we have to rewrite all this? Can it not use our `postcss.config.js`?

Reply:
no :-/ for some reason they wanted different formats. One is for webpack 5 and one is for esbuild.